requirements-dev.txt

```diff
@@ -0,0 +1,15 @@
+pytest==3.2.3
+pytest-bdd==2.18.2
+pytest-cov==2.5.1
+pytest-json==0.4.0
+pytest-mock==1.8.0
+pytest-env==0.6.2
+coverage==4.4.1
+moto==1.3.7
+mock==2.0.0
+pylint==2.2.2
+twine==1.12.1
+placebo==0.8.2
+pyopenssl==19.0.0
+sphinx==1.8.3
```

Comment:
I think we are missing the dependency for the sphinx theme we are using, `sphinx_rtd_theme`

Reply:
Fixed in e2f9bec